src/index.html

```diff
@@ -1,15 +1,174 @@
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+   >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="form"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="Lname">Surname:</label>
+        <input
+         class="input"
+          type="text"
+          name="name"
+          id="Lname"
+          autocomplete="disabled"
+        >
+        <br>
+        <label for="Fname">Name:</label>
+        <input
+          class="input"
+          type="text"
+          name="name"
+          id="Fname"
+          autocomplete="disabled"
+        >
+        <br>
+        <label for="age">How old are You?</label>
+        <input
+          class="input"
+          type="number"
+          name="age"
+          id="age"
+          value="12"
+          min="10"
+          max="100"
+        >
+        <br>
+        <label for="Bdate">Full date of birth:</label>
+        <input
+          class="input"
+          type="date"
+          name="birthday"
+          id="Bdate"
+        >
+        <br>
+        <label for="agreement">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
+          id="agreement"
+          required
+        >
+        <br>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration:</legend>
+        <label for="email">E-mail:</label>
+        <input
+          class="input"
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+       >
+        <br>
+        <label for="password">Password:</label>
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="8"
+          maxlength="20"
+        >
+        <br>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="label">Do you love cats?</div>
+        <input
+          class="input"
+          type="radio"
+          name="yes"
+          id="yes"
+        >
+        <label for="yes">Yes</label>
+        <input
+          class="input"
+          type="radio"
+          name="no"
```

Comment:
```suggestion
          name="cats"
          id="yes"
        >
        <label for="yes">Yes</label>
        <input
          class="input"
          type="radio"
          name="cats"
```

It's the same group of radio buttons, so the name should be the same